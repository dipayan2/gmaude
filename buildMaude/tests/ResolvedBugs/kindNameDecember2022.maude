set show advisories off .

***
***	Test for bug where kinds were printed incorrectly from premodules.
***

fmod FOO{X :: TRIV} is
  sorts S{X} T{X} .
  subsort X$Elt < S{X} T{X} .
  op f : [S{X},T{X}] -> [S{X},T{X}] .
endfm

show mod .
