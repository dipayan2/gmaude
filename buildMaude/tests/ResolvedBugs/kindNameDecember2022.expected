fmod FOO{X :: TRIV} is
  sorts S{X} T{X} .
  subsorts X$Elt < S{X} T{X} .
  op f : [S{X},T{X}] -> [S{X},T{X}] .
endfm
Bye.
